mafia-react/src/components/ErrorDialog.tsx

```diff
@@ -52,5 +58,11 @@ const useStyles = makeStyles((theme: Theme) =>
         marginTop: theme.spacing(2),
       },
     },
+    startpage: {
```

Comment:
i'd name it startPageError
```suggestion
    startPageError: {
```